app/src/main/res/layout/fragment_notification.xml

```diff
@@ -15,8 +15,8 @@
             android:id="@android:id/list"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:dividerHeight="2dp"
-            android:divider="@color/accent"/>
+            android:dividerHeight="0dp"
```

Comment:
http://stackoverflow.com/questions/10163200/how-do-i-remove-the-divider-from-a-listview-on-android